### DISCUSSION TITLE
how to showing show data with video streaming

### DISCUSSION THREAD

**Discussion by yogeshkaharmis** [ant-media/Ant-Media-Server#6092] [FAQs]
hello
I am android developer. in my project I am using ant media sdk for live streaming.
I am RND lots of but I am not find any solution about problem.
my problem is - I am want to show some data with live streaming like data will be over the streaming and data will be send with streaming.
can you please guiding me, how can I get this.
thank you

**Comment by USAMAWIZARD** [comment]
@yogeshkaharmis as far as  I understand you want to share some data while streaming and show it , for sending and receiving data while streaming you can use data channel , here is the [example](https://github.com/ant-media/WebRTC-Android-SDK/blob/master/webrtc-android-sample-app/src/main/java/io/antmedia/webrtc_android_sample_app/basic/DataChannelOnlyActivity.java). 

for displaying a  text on top of video please take a look at [this](https://stackoverflow.com/questions/29914448/how-can-i-add-overlay-images-or-text-on-top-of-videos-in-android).
